packages/launcher-component/src/pickers/repository-picker/repository-picker.tsx

```diff
@@ -8,49 +8,50 @@ import {
   FormSelect,
   FormSelectOption
 } from '@patternfly/react-core';
-
 import { GitInfo } from 'launcher-client';
 
 import { InputProps } from '../../core/types';
 
 export interface RepositoryPickerValue {
   org?: string;
-  name: string;
+  name?: string;
 }
 
-interface RepositoryPickerProps extends InputProps<RepositoryPickerValue | undefined> {
+interface RepositoryPickerProps extends InputProps<RepositoryPickerValue> {
   gitInfo: GitInfo;
+  import?: boolean;
 }
 
 const REPOSITORY_VALUE_REGEXP = new RegExp('^[a-z][a-z0-9-.]{3,63}$');
 
-export const isRepositoryPickerValueValid = (value?: RepositoryPickerValue): boolean => {
-  return !!value && ((!value.org || REPOSITORY_VALUE_REGEXP.test(value.org)) && REPOSITORY_VALUE_REGEXP.test(value.name));
+export const isRepositoryPickerValueValid = (value: RepositoryPickerValue): boolean => {
+  return (!value.org || REPOSITORY_VALUE_REGEXP.test(value.org)) && (value.name !== '' || REPOSITORY_VALUE_REGEXP.test(value.name));
```

Comment:
We need to make something nicer for validation. I made those function valdiator as a starting point, but I don't find them elegant now lol. We need something a bit more complex I guess (that handle required, ...)..

Are you sure about this : `value.name !== '' ||` the `||` feels wrong. I haven't tested.. so maybe I am wrong..